docs/pages/development/getting-started.md

```diff
@@ -67,19 +69,17 @@ Instead, all you need to do to start developing is to run:
 
 <Terminal cmd={["$ npx expo start --dev-client"]} />
 
-and scanning the resulting QR code with your system camera or QR code reader (if you want to develop against a physical device)
-
-or pressing the <kbd>A</kbd> or <kbd>I</kbd> keys (to open the app in your Android or iPhone emulator or simulator respectively).
+Then, scan the resulting QR code with your system camera or QR code reader (if you want to develop it against a physical device). Alternatively, you can press <kbd>a</kbd> or <kbd>i</kbd> keys (to open the app in your Android or iPhone emulator or simulator).
```

Comment:
What does 'it' refer to, the only subject in the sentence is the QR code.

```suggestion
Then, scan the resulting QR code with your system camera or QR code reader (if you want to develop against a physical device). Alternatively, you can press the <kbd>a</kbd> or <kbd>i</kbd> keys (to open the app in your Android emulator or iPhone simulator).
```

Reply:
the `it` is my mistake for overlooking the context.